recipes/r-bigsnpr/meta.yaml

```diff
@@ -38,6 +38,9 @@ requirements:
     - llvm-openmp                  # [osx]
   host:
     - r-base
+    - r-dbplyr
+    - r-dplyr
+    - r-fdrtool
```

Comment:
If these are not in `Depends` nor `Imports` then the intent of the author(s) is presumed to be that they are optional. Since a primary use of Conda packages is for minimal deployments (e.g., a specific step in a processing pipeline) we don't want to add on unnecessary components to installation. Let's move all these to only `test` requirements.

```suggestion
```